lib/generators/sufia/work/templates/locale.en.yml.erb

```diff
@@ -1,4 +1,10 @@
 en:
+  sufia:
+    select_type:
+      <%= file_name %>:
+        name:               "<%= human_name.titleize %>"
+        description:        "<%= human_name %> works"
```

Comment:
Is there a place this can be documented? I fear expectations like this that aren't collected may become barriers to entry for new users.